.github/workflows/ci_doctests.yaml

```
@@ -40,32 +40,31 @@ jobs:
           # fetch all history so that setuptools-scm works
           fetch-depth: 0
 
-      # Install Mambaforge with conda-forge dependencies
-      - name: Setup Mambaforge
-        uses: conda-incubator/setup-miniconda@v2.2.0
+      - name: Setup Micromamba
+        uses: mamba-org/provision-with-micromamba@v15
         with:
-          activate-environment: pygmt
-          python-version: '3.11'
+          environment-name: pygmt
+          environment-file: false
           channels: conda-forge,nodefaults
-          channel-priority: strict
-          miniforge-version: latest
-          miniforge-variant: Mambaforge
-          mamba-version: "*"
-          run-post: false
-          use-mamba: true
-
-      # Install GMT and other required dependencies from conda-forge
-      - name: Install dependencies
-        run: |
-          mamba install gmt=6.4.0 numpy \
-                        pandas xarray netCDF4 packaging \
-                        build make 'pytest>=6.0' \
-                        pytest-cov pytest-doctestplus pytest-mpl sphinx-gallery \
-                        contextily geopandas ipython rioxarray
-
-      # Show installed pkg information for postmortem diagnostic
-      - name: List installed packages
-        run: mamba list
+          extra-specs: |
+            python=3.11
+            gmt=6.4.0
+            numpy
+            pandas
+            xarray
+            netCDF4
+            packaging
+            contextily
+            geopandas
+            ipython
+            rioxarray
+            build
+            make
+            pip
+            pytest-doctestplus
+            pytest-mpl
+            pytest
```

Sort alphabetically. Also, no need for `pytest-cov` here?
```suggestion
            pytest
            pytest-doctestplus
            pytest-mpl
```